src/jit/gentree.cpp

```diff
@@ -12002,16 +12002,22 @@ void Compiler::gtDispTreeRange(LIR::Range& containingRange, GenTree* tree)
 //
 // Arguments:
 //    node - the LIR node to dump.
+//    prefixMsg - an optional prefix for each line of output.
 //
-void Compiler::gtDispLIRNode(GenTree* node)
+void Compiler::gtDispLIRNode(GenTree* node, const char* prefixMsg /* = nullptr */)
 {
-    auto displayOperand = [](GenTree* operand, const char* message, IndentInfo operandArc, IndentStack& indentStack) {
+    auto displayOperand = [](GenTree* operand, const char* message, IndentInfo operandArc, IndentStack& indentStack, const char* prefixMsg = nullptr) {
         assert(operand != nullptr);
         assert(message != nullptr);
 
+        if (prefixMsg != nullptr)
+        {
+            printf("%s", prefixMsg);
```

Comment:
The issue is that the prefixMsg is output for both the operand (here), and the operator (below), and the operator doesn't have any leading space. And, I want the operator and operands to be indented by the same amount, when a prefixMsg is given.

Here's an example from codegen:
```
Generating: N037 (  1,  4) [000034] ------------       t34 =    const(h)  int    0x8B6604 ftn REG NA
Generating:                                                  /--*  t34    int
Generating: N039 (  3,  6) [000035] ------------       t35 = *  indir     int    REG NA
Generating:                                                  /--*  t35    int    control expr
Generating: N041 ( 69, 33) [000008] --C-G-------             *  call      void   System.Console.WriteLine $VN.Void
```
A "full dump" looks like:
```
                                                 /--*  t32    int    control expr
N029 ( 45, 22) [000005] --C-G-------        t5 = *  call      float  Program.f2 $c3
                                                 /--*  t5     float
N031 ( 49, 25) [000021] DAC-G-----L-             *  st.lclVar float  V02 tmp1         d:2 REG NA
N033 (  3,  2) [000022] ------------       t22 =    lclVar    float  V02 tmp1         u:2 (last use) REG NA $c3
                                                 /--*  t22    float
N035 (???,???) [000033] ------------             *  putarg_stk [+0x00] float  REG NA
N037 (  1,  4) [000034] ------------       t34 =    const(h)  int    0x8B6604 ftn REG NA
                                                 /--*  t34    int
N039 (  3,  6) [000035] ------------       t35 = *  indir     int    REG NA
                                                 /--*  t35    int    control expr
N041 ( 69, 33) [000008] --C-G-------             *  call      void   System.Console.WriteLine $VN.Void
N043 (  2,  2) [000013] ------------                il_offset void   IL offset: 25 REG NA
N045 (  1,  1) [000011] ------------       t11 =    const     int    100 REG NA $42
                                                 /--*  t11    int
N047 (  2,  2) [000012] ------------             *  return    int    REG NA $1c0
```